extras/paid-courses.md

```diff
@@ -86,6 +86,10 @@
 ### Artificial Intelligence
 
 ### Machine Learning
+Name | Provider
+:-- | :--:
+[Machine Learning by Andrew Ng](https://www.coursera.org/learn/machine-learning) | [Coursera](https://www.coursera.org)
```

Comment:
This is the course in the [main curriculum](https://github.com/open-source-society/computer-science#machine-learning) so we probably don't need it here.